### ISSUE TITLE
Does this work in India ?

### ISSUE THREAD

**Issue by darkyen** [RIAEvangelist/node-dominos-pizza-api#2]

**Comment by RIAEvangelist** [comment]
not sure. I would love to know, please test

**Comment by RIAEvangelist** [comment]
closing due to inactivity

**Comment by darkyen** [comment]
Well I will try now x)

**Comment by RIAEvangelist** [comment]
awesome

**Comment by RIAEvangelist** [comment]
if you can provide some example calls from the Indian dominos we can start integrating them.

**Comment by zubinmehta** [comment]
Any progress on support for india? Here is the link for ordering: https://pizzaonline.dominos.co.in/ 

I can help out if you need anything to get this done.

**Comment by MetalDent** [comment]
Is this issue fixed or it's still not working for Domino's India?

**Comment by madelinecameron** [comment]
There shouldn't be an API key needed @kaushikraghupathruni. It uses a completely open REST API that the site uses. 

This is why some countries can use the library with a simple change of URLs and others can't.

**Comment by Mayankvdd** [comment]
can someone link me to complete api with interface to take a look 
it would be much appreciated cause  i am new to this

**Comment by RIAEvangelist** [comment]
v3.x branch international settings should make this possible now. Still need to work out the endpoints, but everything is baked in to support it.

**Comment by RIAEvangelist** [comment]
v3 now supports international apis.

Please have a look. If you can not find your API endpoints request help again, and we can work on it together. I will need to work with someone in country to make sure the order completes and you get Pizza.

**Comment by RIAEvangelist** [comment]
If anyone would like to contribute the API list as defined in [docs/InternationalSupport.md](https://github.com/RIAEvangelist/node-dominos-pizza-api/blob/master/docs/InternationalSupport.md)

I would be happy to implement it into the code. I do need help from contributors though because I do not know addresses etc to use for testing without traveling to the country on my own. 

Help would be greatly appreciated, and the world of Pizza lovers would thank you.

**Comment by RIAEvangelist** [comment]
@abhyuday-tripathi

Here is the objecct we need populated from [docs/InternationalSupport.md](https://github.com/RIAEvangelist/node-dominos-pizza-api/blob/master/docs/InternationalSupport.md)

Please replace the URLs with the India URLs and I will integrate it into the code after we test.

You can find more details in the link above if you need. 

```js

{
      referer   :"https://order.dominos.nz/en/pages/order/",
      sourceUri :"order.dominos.nz",
          location:{
          find:urls.location.find
      },
      store     : {
          find    : "https://order.dominos.nz/power/store-locator?s=${line1}&c=${line2}&type=${type}",
          info    : "https://order.dominos.nz/power/store/${storeID}/profile",
          menu    : "https://order.dominos.nz/power/store/${storeID}/menu?lang=${lang}&structured=true"
      },
      order     : {
          validate: "https://order.dominos.nz/power/validate-order",
          price   : "https://order.dominos.nz/power/price-order",
          place   : "https://order.dominos.nz/power/place-order"
      },
      track   : "https://order.dominos.nz/orderstorage/GetTrackerData?"
}

```

**Comment by alestor123** [comment]
Ok does it support delivery in  india @RIAEvangelist and also we have to convert the price to Indian rupee

**Comment by RIAEvangelist** [comment]
If you add in the URLs, everything should automatically be done in Rupees.

**Comment by bucharitesh** [comment]
Is it working for Domino's india ? Did anyone try it ?

**Comment by RIAEvangelist** [comment]
I would be happy to help if anyone would be willing to work with me on it.
I can not order a Pizza in India myself because I am not there.

Anyone who wants to help, please, just let me know. I will work with you
for free and all your friends in India will think you are so cool.

I'm just too busy doing my regular work to stop and do this all by myself.

Namaste

**Comment by RIAEvangelist** [comment]
Also, I already documented how to do the internationalization on the
readme. Please read it before you reach out ;)

**Comment by rishabhkeshan** [comment]
Hey in India, we get an OTP based on our debit card number, how will we support that. @RIAEvangelist I am up for working on the integration and testing.

**Comment by RIAEvangelist** [comment]
In hospital for a couple days will respond when I am out.

**Comment by RIAEvangelist** [comment]
@rishabhkeshan I do not know what an OTP is.

**Comment by bucharitesh** [comment]
Hey why don't we test COD first because Domino's had severe multiple data breaches. Me and my friends are ready to integrate and work with you on this. Plus the testing part.

**Comment by gems2000** [comment]
OTP is One-Time Password. It works as a 2FA for payments and authentication platforms.

**Comment by RIAEvangelist** [comment]
Awesome, Thank you.

Perhaps sometime we could do a remote desktop session, so we could work on
getting India setup

**Comment by rishabhkeshan** [comment]
Yeah I would too like to help out with it, can decide on a suitable time next week

**Comment by abhyuday-tripathi** [comment]
I think we should try Cash on Delivery first

**Comment by RIAEvangelist** [comment]
@rishabhkeshan and @abhyuday-tripathi I think trying CoD first is a good option, do you need help trying this?
